Src/ILGPU/Runtime/Cuda/CudaIpcMemoryBuffer.cs

```diff
@@ -0,0 +1,173 @@
+﻿// ---------------------------------------------------------------------------------------
+//                                        ILGPU
+//                        Copyright (c) 2017-2023 ILGPU Project
```

Comment:
ISSUE: Need to fix up the copyright header. Should be 2024 only.

Reply:
Done, I also assumed you probably wanted the same for all other files I touched.